modeshape-jcr/src/main/java/org/modeshape/jcr/cache/document/WritableSessionCache.java

```diff
@@ -931,8 +933,8 @@ public void save( Set<NodeKey> toBeSaved,
 
         } catch (RuntimeException e) {
             throw e;
-        } catch (Exception e) {
-            throw new WrappedException(e);
+        } catch (Throwable t) {
```

Comment:
@okulikov: just saw this: catching `Throwable` here means `RuntimeException` gets wrapped, and we don't want that. So the code should be reverted to `Exception` (same in the other places as well)